src/qml/Extensions.qml

```diff
@@ -44,8 +44,15 @@ Rectangle {
 
   TextField {
     id: searchBox
-    placeholderText: qsTr("Search")
+    placeholderText: qsTr("   Search")
```

Comment:
instead of putting spaces before the text, you should add anchors and add a left margin to it. probably align it with the icons. this applies to all the other text fields as well.

Reply:
ah alright